src/mig/modules/agentdestroy/agentdestroy.go

```diff
@@ -22,7 +22,7 @@ import (
 func init() {
 	mig.RegisterModule("agentdestroy", func() interface{} {
 		return new(Runner)
-	})
+	}, false)
```

Comment:
That's right. I need to finish converting the other modules, I just implemented the changes in `example` for now as a proof of concept.